lecture-notes/1-cli.md

```diff
@@ -210,25 +197,25 @@ arrow will bring you back, and you can press <kbd>Ctrl-c</kbd> to go straight
 to a fresh prompt no matter how far back you are.
 
 Tab completion is how you ask the shell to figure out what you want to type
-before you're done typing it. For example, let's say you want to list files in
-this course's directory, `/comp/50ISDT/`, on the Tufts homework server. Type
```

Comment:
Can we keep this example if you've set up a server with `/comp/50ISDT/` on it for Notheastern?

Reply:
I guess it still won't be as good, since there won't be multiple candidates for completion.